man/os-release.xml

```diff
@@ -344,8 +344,8 @@
     identification data available to applications by providing the host's
     <filename>/etc/os-release</filename> and
     <filename>/usr/lib/os-release</filename> as respectively
-    <filename>/run/host/etc/os-release</filename> and
-    <filename>/run/host/usr/lib/os-release</filename>.</para>
+    <filename>/run/host/etc-os-release</filename> and
+    <filename>/run/host/usr-lib-os-release</filename>.</para>
```

Comment:
I'm not sure if that is an improvement. In particular, still having two files seems pointless. os-release(5) says " Applications should check for the former, and exclusively use its data if it exists". So since we provide `etc-os-release`, the other file should never be used. In that case, I think we should just the the unification ourselves and name the file `/run/host/os-release`.